examples/stable/test/end_to_end/candid_rpc/workspaces/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "dependencies": {
+        "azle": "0.30.0"
+    },
+    "scripts": {
+        "pretest": "tsx test/pretest.ts",
+        "test": "AZLE_CHECK_GLOBAL_STATE=false jest"
```

Comment:
Do we need to disable the global state checks?

Reply:
yes, or else I wouldn't have done it

Reply:
We can look at if you want, I don't remember exactly the error, but my memory is that it had to do with the weird layout of things. I was thinking that for the sake of expediency we wouldn't need to tests these since they aren't doing anything weird with global state or any state for that matter, and we have so many other good tests, but if you want to take it look at it we can

Reply:
Yes let's take a look